how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-expressions.ipynb

```diff
@@ -83,6 +85,13 @@
     "## Set up a schedule with pipeline parameter expressions"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information about the difference between pipeline and pipeline endpoint, please click [here](https://github.com/Azure/MachineLearningNotebooks/blob/467630f95583a88b731ebc4bdefff1cc525df054/how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-setup-schedule-for-a-published-pipeline.ipynb)."
```

Comment:
Offline sync with Jia, let's move it to the first line of 'Schedule a published pipeline endpoint with pipeline parameter expressions' section.